src/dune_lang/dune_lang.ml

```diff
@@ -1048,7 +1048,7 @@ module Decoder = struct
   [@@inline never]
 
   let fields_mutually_exclusive
-        ?on_dup fields ((Fields (loc, _, _) : _ context) as ctx) state =
+        ?on_dup ?default fields ((Fields (loc, _, _) : _ context) as ctx) state =
```

Comment:
Could you document this parameter in the mli?

Reply:
I think I did already!